crawl-ref/source/dat/strings/de/mutations.txt

```diff
@@ -147,10 +147,10 @@ Your actions are slow, but your melee attacks are powerful.
 Deine Aktionen sind langsam, aber deine Nahkampfangriffe sind mächtig.
 %%%%
 torment resistance 1
-Qualresistenz 1
+Peinigungresistenz 1
```

Comment:
Should it be Peinigungresistenz or Peinigungsresistenz?

Reply:
Peinigungsresistenz